apollo-ios/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## v1.21.0
+
+### New
+- **Enhanced Client Awareness ([#638](https://github.com/apollographql/apollo-ios-dev/pull/638)):** By default send Apollo iOS library name and version as metadata with each network request. 
```

Comment:
Give me a few minutes to wordsmith another description here; something with less potential for controversy.